app/assets/src/components/common/DetailsSidebar/BulkDownloadDetailsMode/DetailsTab.jsx

```diff
@@ -55,7 +55,7 @@ export default class DetailsTab extends React.Component {
           )}
           <FieldList fields={fields} />
         </Accordion>
-        {bulkDownload.pipeline_runs && (
+        {bulkDownload.pipeline_runs.length > 0 && (
```

Comment:
We are sure that pipeline_runs is never undefined, right?

Reply:
Yes, previously pipeline_runs was used without any checks: https://github.com/chanzuckerberg/idseq-web/blob/cdcde0dc35cded0cec3a7d51f9d944f7bbdf3454/app/assets/src/components/common/DetailsSidebar/BulkDownloadDetailsMode/DetailsTab.jsx#L59
So I assume this is fine